eahub/profiles/migrations/0001_initial.py

```diff
@@ -80,6 +80,17 @@ class Migration(migrations.Migration):
                     "available_as_speaker",
                     models.BooleanField(blank=True, default=None, null=True),
                 ),
+                (
```

Comment:
Unless we're going to do another data wipe, we can't change old migrations, we have to create and run a new one. This is what the `makemigrations` command does.